cmd/sqlflowserver/main_test.go

```diff
@@ -273,6 +274,66 @@ FROM iris.predict LIMIT 5;`
 	}
 }
 
+// CaseTrainCustomModel tests using customized models
+func CaseTrainCustomModel(t *testing.T) {
+	a := assert.New(t)
+	trainSQL := `SELECT *
+FROM iris.train
```

Comment:
Fix the indentation?

Reply:
Hi @hungry1526, string constant is supposed to be indented like this. :) Otherwise, the statement will become
```
SELECT *
    FROM iris.train
    ...
```

Reply:
@hungry1526 Thanks for review, sorry for the late reply due to I was debugging some errors in this PR. As @tonyyang-svail commented, this indentation is expected, so I'll keep it.

Reply:
thanks guys.